RELEASE_NOTES

```diff
@@ -3,6 +3,64 @@ For more detailed information, please see the git log.
 
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
+v2.7.0 (March 18th 2016)
+------------------------
+
+feature + bugfix release
+- stabilize Cray support
+    - enable 'dynamic' toolchain option by default for Cray* toolchains (#1581)
+    - remove FFTW from the Cray Toolchain Definition (#1585)
```

Comment:
not sure what you mean here...

We were basically forced to exclude FFTW from the Cray toolchains, because we were running into problems with the Cray-provided `fftw` module is it's always around.

And `libsci` *is* already part of the Cray toolchains.